docs/guides/how-to-wire-up-browsersync-and-gulp-in-my-lando-app.md

```diff
@@ -0,0 +1,170 @@
+---
+title: How to wire up Browsersync and Gulp in my Lando app
+metaTitle: How to wire up Browsersync and Gulp in my Lando app | Lando
+description: How to setup Lando with Browsersync & Gulp so that edits to scss files are reflected immediately in your browser.
+summary: How to setup Lando with Browsersync & Gulp so that edits to scss files are reflected immediately in your browser.
+date: 2020-04-07T14:05:33.088Z
+original: 
+repo: 
+
+author:
+  name: Jonathan Shaw
+  pic: https://www.gravatar.com/avatar/ba46283b2033d3a0201f2042746f81e7
+  link: https://github.com/jonathanjfshaw
+
+feed:
+  enable: true
+  author:
+    - name: Jonathan Shaw
+      email: alliance@lando.dev
+      link: https://github.com/jonathanjfshaw
+  contributor:
+    - name: Jonathan Shaw
+      email: alliance@lando.dev
+      link: https://github.com/jonathanjfshaw
+---
+
+# How to wire up Browsersync and Gulp in my Lando app
+
+<GuideHeader test="" name="Jonathan Shaw" pic="https://www.gravatar.com/avatar/ba46283b2033d3a0201f2042746f81e7" link="https://github.com/jonathanjfshaw" />
+<YouTube url="" />
+
+[Browsersync](https://www.browsersync.io) is a nodejs package that synchronizes website changes to the browser. 
+While it can be used for many things, this guide shows how to use it to push changes made to css files to the browser. 
+
+This guide also assumes you're using Gulp to automate your theming tasks, and are editing SASS/SCSS files that need compiling
+into css before they can be previewed in your browser.
+
+## Configuring Lando for Browsersync
+
+### Create the proxy
+
+First you must create a proxy domain for browsersync to use. Changes you make to your scss files will only show up when
+you are looking at this https://bs.myproject.lndo.site, not at the usual domain https://myproject.lndo.site. 
+Add your equivalent of this to your .lando.yml file:
+```yaml
+proxy:
+    - bs.myproject.lndo.site:3000
+```
+
+### Stream changes whenever Lando is running
+It's most convenient if Browsersync is running in the background whenever your Lando site is up. 
+One simple way to achieve this is to fire off a gulp task whenever Lando starts up. 
+If you had a gulp task called 'watch' then you'd put this in .lando.yml:
+```yaml
+services:
+  node:
+    type: node
+    command: cd /app/path/to/gulp/package/gulp watch
```

Comment:
for this i needed:

```yaml
commmand: cd /app/path/to/gupfile && node_modues/.bin/gulp watch
```

Reply:
My bad, that is what I'm actually using. I just (wrongly) thought the other would be OK and forgot to test it.